flash/image/classification/data.py

```diff
@@ -380,3 +444,236 @@ def show_post_tensor_transform(self, samples: List[Any], running_stage: RunningS
     def show_per_batch_transform(self, batch: List[Any], running_stage):
         win_title: str = f"{running_stage} - show_per_batch_transform"
         self._show_images_and_labels(batch[0], batch[0][DefaultDataKeys.INPUT].shape[0], win_title)
+
+
+class ImageClassificationDataSourceCollection(DataSourceCollection):
+    def __init__(self, **data_source_kwargs):
+        super().__init__(
+            data_sources={
+                DefaultDataSources.FIFTYONE: ImageFiftyOneDataSource(**data_source_kwargs),
+                DefaultDataSources.FILES: ImagePathsDataSource(),
+                DefaultDataSources.FOLDERS: ImagePathsDataSource(),
+                DefaultDataSources.NUMPY: ImageNumpyDataSource(),
+                DefaultDataSources.TENSORS: ImageTensorDataSource(),
+                DefaultDataSources.DATAFRAME: ImageClassificationDataFrameDataSource(),
+                DefaultDataSources.CSV: ImageClassificationDataFrameDataSource(),
+                DefaultDataSources.LABELSTUDIO: LabelStudioImageClassificationDataSource(),
```

Comment:
How come none of the other data sources are passed the `kwargs`?

Reply:
They don't requires it right now, but yes they should all receive it. Definitely a good point !